spec/factories/roles.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :role do
+    user
+  end
```

Comment:
You can generate role with a random access level like so
```suggestion
  factory :role do
    user
    access_level { Role.access_levels.keys.sample }
  end
```
Even if you don't need it now, it's a good idea for factories to fill in all model attributes, even if you set them to nil by default
That way you can edit any of them when you need and it also doubles as a list of attributes for that model with example values